upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -184,17 +183,17 @@ if [[ $state_recorded == "0" && $(hostname) == "ncn-m001" ]]; then
     echo "Taking snapshot of current Kubernetes pod states to ${K8S_PODS_SNAPSHOT}"
     kubectl get pods -A -o wide --show-labels > "${K8S_PODS_SNAPSHOT}"
 
-    } >> ${LOG_FILE} 2>&1
-    record_state ${state_name} "$(hostname)"
+    } >> "${LOG_FILE}" 2>&1
+    record_state "${state_name}" "$(hostname)" | tee -a "${LOG_FILE}"
```

Comment:
I'm just curious if there is a reason that `record_state "${state_name}" "$(hostname)"` is outputted to the LOG_FILE in this PR to release/1.3 and not in the PR to main?